packages/connect/tsconfig.lib.json

```diff
@@ -3,6 +3,8 @@
     "compilerOptions": {
         "outDir": "lib",
         "target": "es2019",
+        // note: do not remove tslib from package.json. It might seem that it is not used
+        // but it is required due to importHelpers: true
```

Comment:
Comments in JSON... is this valid still?

Reply:
github does not like it but it pass through our linting  so we should be good. I just wanted to have the string `tslib` somewhere so that we do not remove it from dependencies someday